ejemplo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The Cheese Guardian</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <section class="main-container">
+    <img src="phone.png" alt="foto"  />
+    <div class="vertical-separator"> </div>
+    <img src="tcg_logo.png" alt="foto"  />		
+  </section>
+  <div class="mountain-image"></div>
+  <div class="second-container"> 	
```

Comment:
if you used a ```section``` as an element for the main container use a section for this one too, i would have used ```features-container``` as a class because the problem in using order words like ```second``` forces you to have this as the second container and if you need to add something before this one the class name has no meaning.